### ISSUE TITLE
issues with chatgpt

### ISSUE THREAD

**Issue by thewildwest12** [lencx/Noi#312]
The chatgpt module is not working when i paste the prompt

**Comment by lencx** [comment]
Have you updated [noi-ask](https://github.com/lencx/Noi/tree/main/extensions/noi-ask) to the latest version?

**Comment by thewildwest12** [comment]
Yes sir, I have. still  when i try to prompt all the chatgpt prompt wont show

**Comment by lencx** [comment]
Prompts need to be added in the settings.

<img width="1217" alt="Screenshot 2024-12-29 at 00 27 25" src="https://github.com/user-attachments/assets/90dac180-d16c-4628-879c-71bbd3560388" />
